src/Dockerfile

```
@@ -0,0 +1,354 @@
+
+### End ovmf_builder_tdx ###
+
+### Start vm_json_builder ###
+FROM ubuntu:24.04 AS vm_json_builder
+ARG SP_VM_IMAGE_VERSION
+ARG S3_BUCKET
+COPY --from=image_builder /output /output
+COPY --from=ovmf_builder_sev /buildroot/ovmf/Build/AmdSev/RELEASE_GCC5/FV/OVMF.fd /output/OVMF_AMD.fd
+COPY --from=ovmf_builder_tdx /buildroot/edk2_build/Build/OvmfX64/RELEASE_GCC5/FV/OVMF.fd /output/OVMF.fd
+COPY --from=kernel_builder /output/boot/vmlinuz-6.12.13-nvidia-gpu-confidential /output/vmlinuz
+ADD bundle/files/scripts/generate_vm_json.sh /buildroot/files/scripts/
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)